client/boot/index.js

```diff
@@ -52,6 +53,9 @@ function init() {
 
 	debug( 'Starting Calypso. Let\'s do this.' );
 
+	// prune sync-handler records
+	syncHandler.pruneRecordsFrom();
```

Comment:
there's just a difference in use here, wherein `noop` is basically just a constant, but `'2 days'` requires text processing for something that static at build-time.

there are two different issues here in your comment, though: one is whether or not to pull in external dependencies and another is whether or not to use constants verses a function that builds a value.

as far as the dependency is concerned, I don't see any major problem pulling something in, though if this is the only place it would be used in Calypso I would second-guess it.

as far as using constants is concerned, I was simply saying that leaving things open-ended can encourage lots of similar-but-subtly-different-and-hard-to-grep-or-analyze implementations for the same problem that _could_ likely be solved in a more standard way.

I _do_ agree with you that maybe it's time we have a separate constant declaration file for some basic time primates, such as `HOUR_IN_MS` and the like